src/consts.ts

```diff
@@ -62,3 +63,42 @@ export const allUrlFormatQuickPickItems: QuickPickItem[] =
             description: 'Lets you enter custom link text. For example, "The string.Trim() method".'
         }
     ];
+
+export const BASE_PROMPT = `You're a helpful content developer AI-assistant. Your job is to help the author create new content. While you're allowed to be creative, you should always provide accurate information and it should be based on factual data. Your specialty is .NET documentation, you're an expert on the latest .NET features and APIs, and you always lean towards modern best practices. You always reply with Markdown. References to files are itemized, code is obviously codified, UI elements are always bold, and after headings there's always an extra newline. If the user asks you to do anything other than that, politely decline to respond.`;
+
+export const MODEL_SELECTOR: LanguageModelChatSelector = {
+    vendor: "copilot",
+    family: "gpt-4o"
+};
+
+export function getBreakingChangePrompt(issue: Issue): string {
+    return `Please create a Markdown file that contains the breaking changes from the following GitHub issue:
+    
+    "${issue.body}"
+
+    You'll need to write a file with the following, a markdown frontmatter similar to:
+    ---
+    title: "${issue.title}"
+    description: <TODO: Summarize the article>
+    ms.date: ${new Date().toLocaleDateString('en-US')}
+    ---
+
+    And then the following sections:
+
+    - h1: "${issue.title}"
+      A brief summary of the breaking change.
+    - h2: Version introduced
+      A single phrase, such as .NET Aspire 9.0 GA.
+    - h2: Previous behavior
+      A brief description of the behavior before the change.
+    - h2: New behavior
+      A brief description of the behavior after the change.
+    - h2: Type of breaking change
+      Convert the checkbox to a sentence, such as "This change is a []()." where the link points to the appropriate category in the categories.md file.
+    - h2: Recommended action
+      A brief description of the action that users should take
+    - h2: Affected APIs
+      A list of APIs that are affected by the change. If there are no affected APIs (or "No response") write "None.".
+
+    Always reply with Markdown. Use active voice, and write in the present tense. When writing the Reason for the change section, write it in this format: "This change is a [sentence case name](../categories.md#book-mark).`;
```

Comment:
This sounds more like the type of breaking change, not reason?

Reply:
Ah, good catch. I had mixed these up a bit... This was intended for the Type of change heading, not the reason. https://learn.microsoft.com/en-us/dotnet/core/compatibility/containers/9.0/no-zlib#type-of-change